app-admin/webmin/webmin-1.791.ebuild

```diff
@@ -285,7 +285,7 @@ pkg_config(){
 	export perl="$( which perl )"
 	export os_type='gentoo-linux'
 	export os_version='*'
-	export real_os_type='Gentoo Linux'
+	export real_os_type="${DISTRO} Linux"
```

Comment:
That looks bad. We certainly shouldn't be forcing 'foo Linux' long form on people. os-release has separate NAME and PRETTY_NAME to account for this.